client-v2/src/keyboard/keyboardActionMaps/move.ts

```diff
@@ -0,0 +1,42 @@
+import Raven from 'raven-js';
+import { KeyboardActionMap, ApplicationFocusStates } from 'keyboard';
+import { Dispatch } from 'types/Store';
+import { keyboardArticleFragmentMove } from 'actions/KeyboardNavigation';
+
+const moveUp: KeyboardActionMap = {
```

Comment:
There is some potential for cleaning up here and removing code duplication, going to look at refactoring when we start doing the collection actions

Reply:
Make sense 👌